server/container_create_linux.go

```diff
@@ -250,25 +234,14 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		}
 		for target, mode := range mounts {
 			if !isInCRIMounts(target, containerConfig.GetMounts()) {
-				mnt := rspec.Mount{
```

Comment:
@haircommander I was thinking I shouldn't add removing the mount in places like this, because this doesn't seem to remove it? but maybe I'm wrong !

Reply:
I think we don't remove it because we don't expect it to be there (it's a target mount in the container's rootfs, `/var/lib/containers/storage/overlay/$randomID....` which would be remarkable if it was already there. I don't think there's any harm in double removing the mount, and I'm pretty sure all calls to AddMount() are meant to replace one that exists. I think calling them together would be best, unless something breaks when we delete if it's not there

Reply:
fixed !